consensus/state_test.go

```diff
@@ -1370,8 +1366,6 @@ func TestResetTimeoutPrecommitUponNewHeight(t *testing.T) {
 	ensurePrecommit(voteCh, height, round)
 	validatePrecommit(t, cs1, round, round, vss[0], theBlockHash, theBlockHash)
 
-	rs = cs1.GetRoundState()
```

Comment:
doesn't this change the behavior?

Reply:
here you aren't using  `cs1.GetRoundState()` until you set it on L1390, it did not change the behavior of the tests